### PR TITLE
Qap 39 new balance size board

### PR DESCRIPTION
**Pull request by Safron1ta2**

### CODE REVIEW

File: src/test/java/Safron/Disney/Pages/DisneyBackToSchoolPageElements.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import BasePageAndTestInit.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class DisneyBackToSchoolPageElements extends BasePage {
+    public DisneyBackToSchoolPageElements(WebDriver driver) {
+        super(driver);
+    }
+    private static final String FIRST_PRODUCT_NAME = "//div[@data-pid='464022334657']//a[@class='product__tile_link']";
```
Comment (AndriiMyronenko418) [inline src/test/java/Safron/Disney/Pages/DisneyBackToSchoolPageElements.java:11]: add empty line before variables

**Review by AndriiMyronenko418 — CHANGES_REQUESTED** [review]

File: src/test/java/BasePageAndTestInit/BasePage.java [hunk 2]
```diff
@@ -43,4 +43,11 @@ public void moveCursor(String locator){
         Actions actions = new Actions(driver);
         actions.moveToElement($(locator)).perform();
     }
+
+    public void doMoveCursor(WebElement element){
```
[the remark this answers is not in the document]
Comment (Safron1ta2) [reply src/test/java/BasePageAndTestInit/BasePage.java]: fixed

File: src/test/java/BasePageAndTestInit/BasePage.java [hunk 3]
```diff
@@ -43,4 +43,11 @@ public void moveCursor(String locator){
...
+        Actions actions = new Actions(driver);
+        actions.moveToElement(element).perform();
+    }
+
```
[the remark this answers is not in the document]
Comment (Safron1ta2) [reply src/test/java/BasePageAndTestInit/BasePage.java]: fixed

File: src/test/java/Safron/Disney/Pages/DisneyBackToSchoolPageElements.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    public WebElement productPriceCart(){
+        return $(PRODUCT_PRICE_CART);
+    }
+    public WebElement productNameCart(){
+        return $(PRODUCT_NAME_CART);
+    }
+    public WebElement addBagProductBtn(){
+        return $(ADD_BAG_PRODUCT_BTN);
+    }
```
[the remark this answers is not in the document]
Comment (Safron1ta2) [reply src/test/java/Safron/Disney/Pages/DisneyBackToSchoolPageElements.java:42]: fixed

File: src/test/java/Safron/Disney/Pages/DisneyBackToSchoolPageElements.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    public WebElement addWishBtn(){
+        return $(ADD_WISH_BTN);
+    }
+
```
[the remark this answers is not in the document]
Comment (Safron1ta2) [reply src/test/java/Safron/Disney/Pages/DisneyBackToSchoolPageElements.java]: fixed

**Review by AndriiMyronenko418 — APPROVED** [review]

**Pull request merged**